src/data.js

```diff
@@ -1215,6 +1215,23 @@ module.exports = [
       'Linux',
     ],
   },
+  {
+    name: 'Keith Donegan',
+    description:
+      'Irish WordPress developer, based in London, UK.',
+    url: 'https://www.keithdonegan.com/uses/',
+    twitter: '@eirestudio',
+    emoji: '🤪',
+    country: '🇮🇪🇸',
```

Comment:
Any reason there's 2 emoji? CI won't pass until this field contains a single flag

Reply:
No idea, I'll change.